### ISSUE TITLE
This script is too good

### ISSUE THREAD

**Issue by ryhaph** [StreisandEffect/streisand#1655]
I just wanted to say thank you for all the work you put in, just baffled by how much is here. Hahaha sorry for my funny way of complimenting the work, I just saw all the issues and thought whoever made this could do with some encouragement. CHEERS! :)

**Comment by TheNomad11** [comment]
Agree! And the install process has become so much easier compared to a few years ago, Thanks a million!
